test/e2e/performanceprofile/testdata/render-expected-output/manual_tuned.yaml

```diff
@@ -61,7 +61,8 @@ spec:
   - machineConfigLabels:
       machineconfiguration.openshift.io/role: worker-cnf
     operand:
-      debug: false
+      tunedConfig:
+        reapply_sysctl: null
```

Comment:
`reapply_sysctl` is a Boolean, why does it have `null` value?

Reply:
You tell me :) This is the value the test is seeing, I only updated the expected value for it to pass as PAO does not care about this part.

Reply:
I might tell you, but I have no idea about the test you're talking about. :)  I guess it is `null` because `tunedConfig` was not specified at all?  Good to know it was manually updated by you.

Reply:
This is the expected value of Tuned when PerfProfile is processed by the render stage. If you check couple of lines above this, we mostly care about the generated tuned profile. The rest is just what NTO structures serialize to by default.